src/containers/PageContainer.jsx

```diff
@@ -122,24 +122,53 @@ class PageContainer extends React.PureComponent {
       questionsByPage,
     };
 
+    const blocksGroups = [
```

Comment:
moved `blocksGroups` up to the parent container so that it only needs to be declared once.